app/controllers/api/v1/users_controller.rb

```diff
@@ -1,20 +1,24 @@
 class Api::V1::UsersController < ApplicationController
   skip_before_action :authenticate_request,
-                     :email_verified?,
+                     :is_email_verified?,
                      only: [:create_patient]
-  before_action :restrict_user, only: %i[index show_user]
-  before_action :set_user, only: [:show_user]
+  before_action :restrict_user, only: %i[show_users show_user]
+  before_action :restrict_patient, only: [:show_patients]
+  before_action :set_user, only: %i[show_user destroy_user]
 
-  def index
-    @users =
-      User.all.select do |user|
-        (user.roles.first == patient_role || user.roles.first == doctor_role)
-      end
+  def show_users
+    @users = User.all.each.select { |user| user.roles.first != admin_role }
```

Comment:
I think its much better to rename this `show_users` method as `users` only? WDYT?

Reply:
same with other `method` here :)